osv/impact.py

```diff
@@ -33,9 +33,10 @@
 TAG_PREFIX = 'refs/tags/'
 
 AffectedResult = collections.namedtuple('AffectedResult',
-                                        'tags commits affected_ranges')
+                                        ['tags', 'commits', 'affected_ranges'])
 
-AnalyzeResult = collections.namedtuple('AnalyzeResult', 'has_changes commits')
+AnalyzeResult = collections.namedtuple('AnalyzeResult',
+                                       ['has_changes', 'commits'])
 
 TagsInfo = collections.namedtuple('TagsInfo', 'tags latest_tag')
```

Comment:
nit: also change this last one to use the list notation

Reply:
I'd probably just convert all of these to dataclasses instead, which are a lot nicer to work with.

Reply:
I've made it consistent for now in 65766cb and I'll look at how invasive a change dataclasses would be if I get a chance. I didn't want to go too much off on a tangent.

Reply:
Converted to dataclasses in 03dac88